README.md

```diff
@@ -1,13 +1,237 @@
 # 💪 strong-config
 
-Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
+<!-- markdownlint-disable line-length -->
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master) [![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+<!-- markdownlint-enable line-length -->
 
-[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Simple, safe and secure config management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
 
-[![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Have you ever struggled with different config values across local, CI,
+and various cluster environments? If no, did you ever forget to update the
```

Comment:
We shouldn't presume they're using clusters here. Just "various environments" is fine.

Reply:
done